tests/toWarnDev.js

```diff
@@ -29,9 +29,12 @@ SOFTWARE.
 /* eslint-disable func-names */
 /* eslint-disable complexity */
 const util = require('util')
-const jestDiff = require('jest-diff').default
+const jestDiff = require('jest-diff').diff
```

Comment:
No idea why this ever worked. It didn't change in `jest-diff` between 29.4.1 and 29.7.0 so it must have been some other thing that made it work incidentally.